docs/core/concepts/tasks.md

```diff
@@ -12,6 +12,8 @@ from prefect import task
 @task
 def plus_one(x):
     return x + 1
+
+plus_one.run(x=2)  # 3
```

Comment:
Do you think it would be better suited in a TIP below the line:
> All Task subclasses must have a run() method.
Which mentions that for testing purposed tasks can be directly called with .run?

Reply:
Yea that makes sense to me 👍